zenith_utils/src/pq_proto.rs

```diff
@@ -892,7 +907,12 @@ impl ZenithFeedback {
                     }
                 }
                 _ => {
-                    info!("ZenithFeedback parse. unknown key {}", key)
+                    let len = buf.get_i32();
+                    info!(
+                        "ZenithFeedback parse. unknown key {} of len {}. Skip it.",
+                        key, len
+                    );
+                    buf.advance(len as usize);
```

Comment:
[advance](https://docs.rs/bytes/0.4.7/bytes/buf/trait.Buf.html#tymethod.advance) takes cnt in bytes, not bits, so I'd pass len as bytes count in serialize(). I also think it's better to add unit tests here with encode/decode in different cases: with and without unknown fields.